sdk/src/Core/Amazon.Runtime/Pipeline/HttpHandler/_netstandard/HttpRequestMessageFactory.cs

```diff
@@ -474,14 +474,7 @@ public void SetRequestHeaders(IDictionary<string, string> headers)
         /// <returns>The request content.</returns>
         public HttpContent GetRequestContent()
```

Comment:
Turning this into a simple `return` is triggering the build error [CA1024](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1024). I don't want to change this public method so can you wrap the method with `#pragma warning disable CA1024` and `#pragma warning restore CA1024`?

Reply:
Done